corehq/util/soft_assert/api.py

```diff
@@ -3,9 +3,18 @@
 from corehq.apps.hqwebapp.tasks import send_mail_async, mail_admins_async
 from corehq.util.global_request import get_request
 from corehq.util.soft_assert.core import SoftAssert
+from django.views.debug import get_exception_reporter_filter
 
 
 def _send_message(info, backend):
+    try:
+        request = get_request()
+        # de-sensitize requests that might have passwords
```

Comment:
It will end up as `None`
````python
In [53]:  get_exception_reporter_filter(None).get_request_repr(None)
Out[53]: 'None'

Is that bad? Otherwise there would be an extra conditional with an extra copy of line 16